nms/app/packages/magmalte/alerts/lteAlerts.js

```diff
@@ -22,15 +22,31 @@ export default function getLteAlerts(
   return {
     'Service Restart Alert': {
       alert: 'Service Restart Alert',
-      expr: `avg_over_time(service_restart_status{networkID=~"${networkID}"}[5m]) > 0`,
+      expr: `increase(service_restart_status{networkID=~"${networkID}"}[5m]) > 0`,
       labels: {severity: 'minor'},
       annotations: {description: 'Alerts upon service restarts'},
     },
     'Cpu Percent Alert': {
       alert: 'Cpu Percent Alert',
-      expr: `avg_over_time(cpu_percent{networkID=~"${networkID}"}[5m]) > 75`,
```

Comment:
How are we picking these percentages?

Reply:
Heuristically, cpu percent greater than 75% seems to be a good indicator that something might be going on.